test/base/MockExportsMovementsConnector.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package base
+
+import connectors.CustomsDeclareExportsMovementsConnector
+import connectors.exchanges.Consolidation
+import models.notifications.NotificationFrontendModel
+import models.requests.MovementRequest
+import models.submissions.SubmissionFrontendModel
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito._
+import org.scalatest.{BeforeAndAfterEach, Suite}
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.concurrent.Future
+
+trait MockExportsMovementsConnector extends MockitoSugar with BeforeAndAfterEach { self: Suite =>
```

Comment:
Do we really need this?

Writing Mockito mocks is intuitive. Easy to remember.

I find these classes just make writing tests more complex as you have to remember a bunch of helper methods exist that just make the code less readable (as people forget what they do so use them when they aren't required)

Reply:
Agree. This is copy from the external service. Will get rid of it.